cartofacade/capi_test.go

```diff
@@ -229,9 +264,23 @@ func TestCGoAPI(t *testing.T) {
 		test.That(t, position.Kmag, test.ShouldNotEqual, 0)
 		test.That(t, position.Real, test.ShouldNotEqual, 1)
 
-		// test getInternalState
-		_, err = vc.getInternalState()
-		test.That(t, err, test.ShouldResemble, errors.New("nil internal state"))
+		// test getPointCloudMap returns non 0 response
+		// on aarm64 linux this always returns a different response
```

Comment:
Confirming "different" means "different from the PCD returned after the 2nd lidar reading was added", not "different each time the test is run"?

If so would you mind clarifying in the text?

Reply:
done